### ISSUE TITLE
Main Page(Activity)

### ISSUE THREAD

**Issue by Akshima-Ghai** [Akshima-Ghai/OneEducationalAppForAll#11]
Next Page which will open after the splash screen, try to make it look professional. This page will include the introduction abt the app, will also include different sections for video lectures, Blogs, TestSeries, Doubt assistance, try to check creative ideas from dribble, and include every possible feature you can think of.

**Comment by Anup0099** [comment]
I would like to work on this issue.

**Comment by Akshima-Ghai** [comment]
@Anup0099 issue assigned, show your creativity to the best. ALL THE BEST

**Comment by Anup0099** [comment]
@Akshima-Ghai  I have made this. After the Splash screen, I have made two screens. Is it ok?

https://user-images.githubusercontent.com/75351645/124282409-6d31e400-db68-11eb-8d17-8efc0a021943.mp4

![main-page](https://user-images.githubusercontent.com/75351645/124282445-758a1f00-db68-11eb-8476-8817ca77ff18.jpeg)
![user-page](https://user-images.githubusercontent.com/75351645/124282456-77ec7900-db68-11eb-8bc3-14ea889a306d.jpeg)

**Comment by Akshima-Ghai** [comment]
@Anup0099 good designs 
1)Make sure the app is responsive 
2) add some fonts and design to "hello students"
3) The boxes on the second page and not perfectly oriented 
Try to complete this work and we can later see if some more changes needed to be made or not 
Great Work, Keep it up, try improving the updates mentioned

**Comment by Anup0099** [comment]
@Akshima-Ghai  Should I make it like if the user clicks on any of the box it will open in the new screen?

**Comment by Akshima-Ghai** [comment]
Yes @Anup0099, that's the exact same way we wanted it to be, but first, update the changes mentioned and also keep a scope for adding more boxes in the future if needed. GOOD LUCK

**Comment by Anup0099** [comment]
Ok

**Comment by Anup0099** [comment]
I am not able to make the app responsive. When I see it horizontally it changes. Also, I have changed the Hello students text. And the orientations, whenever I am changing it's giving me overflow.
![pic2](https://user-images.githubusercontent.com/75351645/124346997-7f129600-dbff-11eb-86d0-f873aa4c7599.jpeg)

**Comment by Akshima-Ghai** [comment]
@Anup0099 discuss your doubts on the grp, and improve your code , then we will merge it

**Comment by amartyanambiar** [comment]
> @Akshima-Ghai I have made this. After the Splash screen, I have made two screens. Is it ok?
> 
>  video.3.mp4 
> ![main-page](https://user-images.githubusercontent.com/75351645/124282445-758a1f00-db68-11eb-8476-8817ca77ff18.jpeg)
> ![user-page](https://user-images.githubusercontent.com/75351645/124282456-77ec7900-db68-11eb-8bc3-14ea889a306d.jpeg)

Nice!

**Comment by Akshima-Ghai** [comment]
@Anup0099 pls create a pr so that we can add these changes

**Comment by Anup0099** [comment]
@Akshima-Ghai  I have created the pr.
